spec/rubocop/cop/cop_spec.rb

```diff
@@ -132,4 +132,34 @@
       end
     end
   end
+
+  describe '#autocorrect?' do
+    # dummy config for a generic cop instance
+    let(:config) { RuboCop::Config.new({}) }
+    let(:options) { nil }
+    let(:cop) { described_class.new(config, options) }
+    let(:support_autocorrect) { true }
+    subject { cop.autocorrect? }
+
+    before do
+      allow(cop).to receive(:support_autocorrect?) { support_autocorrect }
+    end
+
+    context 'when the option is false' do
+      let(:options) { { auto_correct: false }}
+      it { should be_falsey }
```

Comment:
Since the method always returns a boolean, you could use `be false` and `be true` instead of `be_falsey` and `be_truthy`.